readme.md

```diff
@@ -240,6 +240,43 @@ being merged. The "Merge commit" and "Squash and merge"
 [strategies](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#about-merge-queues)
 are currently supported. "Rebase and merge" does not yet work correctly.
 
+## Automatic upgrades
```

Comment:
let's also add a note saying that service integration for this is coming, so that users won't have to do this by themselves